components/calendar.scss

```diff
@@ -0,0 +1,7 @@
+$fd-icons-path: "../scss/icons/";
+$fd-fonts-path: "../scss/fonts/";
+
+@import "../scss/components/calendar";
+@import "../scss/icons/icon";
+@import "../scss/fonts/72";
```

Comment:
Icons are included because of this line: https://github.com/SAP/fundamental-styles/blob/master/scss/components/calendar.scss#L155

Although, it might be fine without them. Thoughts @greg-a-smith ?

Reply:
@jbadan Definitely some interesting CSS. 😄 Although that method of flipping the arrow icons is used (creative), if you check the rendered markup, there is actually a button component (class `fd-button`) that the calendar component will use internally.  Since button already comes with icons, they should be left out of calendar.